public/assets/js/publishFunctions.js

```diff
@@ -65,41 +65,87 @@ function validateClaimName (name) {
 	});
 	return deferred;
 }
+// validation function which checks all aspects of the publish submission
```

Comment:
@98farhan94 this function is run when the submit button is clicked to submit the file for publishing.